cmd/gobgp_exporter/main.go

```diff
@@ -44,6 +53,28 @@ func main() {
 		Timeout: pollTimeout,
 	}
 
+	if serverTLS {
+		opts.TLS = new(tls.Config)
+		if len(serverTLSCAPath) > 0 {
+			// assuming PEM file here
+			pemCerts, err := os.ReadFile(serverTLSCAPath)
```

Comment:
```suggestion
			pemCerts, err := os.ReadFile(filepath.Clean(serverTLSCAPath))
```
for path sanitization

Reply:
Done!

Reply:
I believe you also need to `import "filepath"`

Reply:
Oh yep, I was a bit too fast there and didn't run goimports against this. Sorry about that!